### PR TITLE
Added setWifiState

### PR DESCRIPTION
**Pull request by sravanmedarapu**
depends on 

- #https://github.com/appium/appium-uiautomator2-server/pull/42

 

- #https://github.com/appium/appium-android-driver/pull/196

Fix for appium/appium#6615
cc: @jlipps, @imurchie

### CODE REVIEW

File: lib/commands/network.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+
+let extensions = {}, commands = {};
+
+commands.setWifiState = async function(wifi) {
+  return await this.uiautomator2.jwproxy.command('/appium/device/toggle_wifi', 'POST', {wifi});
```
Comment (sravanmedarapu) [inline lib/commands/network.js]: Seems network connection API is not stable, users reported issue https://github.com/appium/appium/issues/6615

**Comment by jlipps** [comment]
@sravanmedarapu does this pass lint? I think I see a lint error. Also, why is this test being skipped on CI?

**Comment by sravanmedarapu** [comment]
@jlipps, Emulator not showing up in given 15000 ms
```
info android-emu Waiting 15000 ms for emu...
info android-emu 
killing process with command:sudo pkill -f emulator
[10:41:21] Emulator didn't start properly, error:
````
Regarding Linting issue, I couldn't see any issue either in [travis](https://travis-ci.org/appium/appium-uiautomator2-driver/builds/181686528) or locally. Please let me know if have any specific case.

**Comment by jlipps** [comment]
@sravanmedarapu i guess lint is OK then. but let's not skip a particular test if CI is failing in general, though, right? otherwise we might fix the emulator and forget to un-skip this test later?

**Comment by sravanmedarapu** [comment]
@jlipps reason for skipping spec is WIFI toggle functionality not working on default emulators. To make sure nothing broken, executed skipped test locally working fine.

**Comment by jlipps** [comment]
👍

**Pull request merged**